addons/editor-theme3/addon.json

```diff
@@ -367,32 +436,10 @@
         "text": "white"
       }
     },
-    {
-      "name": "Dark",
-      "id": "dark",
-      "description": "Dark versions of the default colors",
-      "values": {
-        "motion-color": "#004099",
-        "looks-color": "#220066",
-        "sounds-color": "#752475",
-        "events-color": "#997300",
-        "control-color": "#664100",
-        "sensing-color": "#1f5f7a",
-        "operators-color": "#235c23",
-        "data-color": "#b35900",
-        "data-lists-color": "#993300",
-        "custom-color": "#99004d",
-        "Pen-color": "#064734",
-        "sa-color": "#166966",
-        "comment-color": "#423f24",
-        "input-color": "#202020",
-        "text": "white"
-      }
-    },
     {
       "name": "Black",
       "id": "black",
-      "description": "Makes block backgrounds black",
+      "description": "Makes block backgrounds black and borders/text colored",
```

Comment:
Could we call this preset "Neon"?

Reply:
The chemical element?

Reply:
> The chemical element?

Yeah. They just look reminiscent of neon lights to me.

Reply:
It might be a bit more confusing. But at least it's not hard to translate into other languages, since it's just the name of a chemical element.

Reply:
Both names self-explain different aspects, so it's hard to choose which one to use.

Reply:
Why not "neon black"? :P

Reply:
Oh, that's a good option.

Reply:
I'm not sure if that's a very good name - the main purpose of the preset is to make the background darker, not the text brighter.